config/default.json

```diff
@@ -96,14 +96,20 @@
       "foreignAddress": "0x8F086f895deBc23473dfe507dd4BF35D6184552c",
       "symbol": "ANT",
       "decimals": 3
+    },
+    {
```

Comment:
This should not be added to default.json, we should change the config for beta on its server

Reply:
Yes you are right but for test in local environment it's better to be in default.json
Are you sure for reverting these changes?

Reply:
Yes, it's for your testing and should not be committed to the develop branch.

Reply:
I removed changes from default.json